spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -491,7 +472,6 @@ def check_response_error(code)
       it_behaves_like 'it issues errors when sul_author_id does not exist'
       it_behaves_like 'it issues errors for cap_profile_id'
       it_behaves_like 'it checks author for the correct cap_profile_id'
-      it_behaves_like 'it checks author for a cap_profile_id'
       it_behaves_like 'it handles invalid authorship attributes'
 
```

Comment:
It's possible these specs are still testing useful behavior for the API, although the implementation has changed.  Something like 'it validates cap_profile_id' and other such behavior is still relevant at the API level?  Did the new validation break these specs?  Could they be adjusted to continue testing the API level?  Am I missing something here - is this behavior still tested somewhere?  Is there any new behavior introduced - does it now trigger any new error responses (either deliberately or through rails/grape magic)?

Reply:
Please remember that I am following up with larger changes to the API.  These tests are excluded because the **setup** is a scenario that is now impossible:
```
author.cap_profile_id = ''		
author.save!
```
The whole idea here is that the model guarantees a `cap_profile_id`.

Reply:
👌 - I see, and other specs check for errors related to cap-profile-id.